contributors/guide/README.md

```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
+- [Contributor Playground](#contributor-playground)
+- [Contributor Workshops](#contributor-workshops)
+- [Community](#community)
+- [Communication](#communication)
+- [Events](#events)
+- [Meetups](#meetups)
+- [Mentorship](#mentorship)
+- [Advanced Topics](#advanced-topics)
+
+# Getting Started
+
+## Welcome
 
 Have you ever wanted to contribute to the coolest cloud technology?
-We will help you understand the organization of the Kubernetes project and direct you to the best places to get started.
-You'll be able to pick up issues, write code to fix them, and get your work reviewed and merged.
+This guide will help you understand the overall organization of the Kubernetes project, 
+and direct you to the best places to get started contributing. 
+You'll be able to pick up issues, write code to fix them, 
+and get your work reviewed and merged.
 
 This document is the single source of truth for how to contribute to the code base.
-Feel free to browse the [open issues](https://github.com/kubernetes/community/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fcontributor-guide) and file new ones, all feedback welcome!
+Feel free to browse the 
+[open issues] 
+and file new ones, all feedback is welcome!
 
 ## Contributor Guide
 
 Welcome to Kubernetes! This guide is broken up into the following sections. 
-It is recommended that you follow these steps in order. 
+It is recommended that you follow these steps in order: 
 
 - [Welcome](#welcome) - this page 
-- [Prerequisites](#prerequisites) - things you need to complete before contributing
-- [Your First Contribution](first-contribution.md) - things you need to know before your first contribution
-- [Contributing](contributing.md) - the main guide with contributor information
+- [Prerequisites](#prerequisites) - tasks you need to complete before you can start 
+contributing to Kubernetes
+- [Your First Contribution](first-contribution.md) - things you'll need to know 
+before making your first contribution
+- [Contributing](contributing.md) - the main reference guide to contributing 
+to Kubernetes
 
 ## Resources Available
 
 - [Kubernetes Contributor Playground](#kubernetes-contributor-playground)
-- [Contributor Workshops](#contributor-workshops)
+- [Contributor Workshops](#kubernetes-contributor-workshops)
 - [Community](#community)
   - [Communication](#communication-1)
   - [Events](#events)
     - [Meetups](#meetups)
   - [Mentorship](#mentorship)
 - [Advanced Topics](#advanced-topics)
 
-# Prerequisites
+## Prerequisites
 
-Before submitting code to the project you should first take care of the following prerequisites.
-These steps are checked by a bot during your first submission, so doing these steps first will make your first contribution easier: 
+Before submitting code to Kubernetes, you should first complete the following 
+prerequisites.
+These steps are checked automatically by [a bot]
+during your first submission. 
+Completing these steps will make your first contribution easier: 
```

Comment:
```suggestion
These steps are checked automatically by [a bot] during your first submission.
Completing these steps will make your first contribution easier: 
```
These can be consolidated to match the 80 character guideline. 

There are other instances like this in the doc, I'd consolidate where needed.

Reply:
Done!